doc/configuring.rst

```diff
@@ -48,7 +48,7 @@ Both `name` and `command`, however, can be undefined. A build step without a nam
 a build step without a command will do nothing, but will still appear in log (and have a step number).
 
```

Comment:
to line 6: please add a link to 'Project configuration structures' when referencing configuration file.

Reply:
Please check whether the proposed change is satisfactory.